lib/Segments/SegmentListingRepository.php

```diff
@@ -50,7 +52,7 @@ protected function applyFilters(QueryBuilder $queryBuilder, array $filters) {
   }
 
   protected function applyParameters(QueryBuilder $queryBuilder, array $parameters) {
-    $types = [SegmentEntity::TYPE_DEFAULT, SegmentEntity::TYPE_WP_USERS];
+    $types = $this->types;
     if ($this->wooCommerce->shouldShowWooCommerceSegment()) {
       $types[] = SegmentEntity::TYPE_WC_USERS;
     }
```

Comment:
The listing repository is used for both static and dynamic segments and WooCommerce segment is added dynamically based on WooCommerce plugin status. This causes that when WooCommerce plugin is active we display WooCommerce segment in both static and dynamic segment lists